packages/python-packages/apiview-copilot/main.tsp

```diff
@@ -0,0 +1,123 @@
+// main.tsp - TypeSpec API surface for apiview-copilot FastAPI app
+
+using TypeSpec.Rest;
+using TypeSpec.OpenAPI3;
+
+@service({
+  title: "APIView Copilot Service",
+  version: "1.0.0"
+})
+@useDependency(RestResource)
+@useDependency(OpenAPI3Resource)
+
+namespace ApiViewCopilot {
```

Comment:
If we want to do versioning, like the version `1.0.0` then we need to version the namespace and list the version
```tsp
@versioned(Versions)
namespace ApiViewCopilot;

  enum Versions {
  `1.0.0`,
}

```

Reply:
For now, I'll keep it simple and not having versioning.